### ISSUE TITLE
How do I update miniforge?

### ISSUE THREAD

**Issue by arturdaraujo** [conda-forge/miniforge#674]
### Comment:

My currently mamba version installed is 1.5.8.

I received a notification about an update, but I'm unsure if I should proceed with this:

`conda update -n base -c conda-forge conda
`

I see that the lastest Miniforge version is currently at 24.9.0, while the latest conda version is 24.9.2 (I believe this would update the wrong "conda").

The only documentation I found for updating Miniforge was on micromamba updates https://mamba.readthedocs.io/en/latest/installation/micromamba-installation.html, but I’m not sure if this is the correct approach.

What’s the recommended way to update Miniforge? Should I uninstall it and reinstall the latest version, or is there a simpler method?

```
==> WARNING: A newer version of conda exists. <==
    current version: 24.3.0
    latest version: 24.9.2

Please update conda by running

    $ conda update -n base -c conda-forge conda
```

```
(py311_featurewiz057) artur@LAPTOP-80C5GA6H:~$ mamba info

          mamba version : 1.5.8
     active environment : py311_featurewiz057
    active env location : /home/artur/miniforge3/envs/py311_featurewiz057
            shell level : 3
       user config file : /home/artur/.condarc
 populated config files : /home/artur/miniforge3/.condarc
          conda version : 24.3.0
    conda-build version : not installed
         python version : 3.10.14.final.0
                 solver : libmamba (default)
       virtual packages : __archspec=1=skylake
                          __conda=24.3.0=0
                          __cuda=12.6=0
                          __glibc=2.39=0
                          __linux=5.15.153.1=0
                          __unix=0=0
       base environment : /home/artur/miniforge3  (writable)
      conda av data dir : /home/artur/miniforge3/etc/conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
          package cache : /home/artur/miniforge3/pkgs
                          /home/artur/.conda/pkgs
       envs directories : /home/artur/miniforge3/envs
                          /home/artur/.conda/envs
               platform : linux-64
             user-agent : conda/24.3.0 requests/2.31.0 CPython/3.10.14 Linux/5.15.153.1-microsoft-standard-WSL2 ubuntu/24.04 glibc/2.39 solver/libmamba conda-libmamba-solver/24.1.0 libmambapy/1.5.8
                UID:GID : 1002:1002
             netrc file : None
           offline mode : False

```

**Comment by hmaarrfk** [comment]
The suggested command in the terminal should be correct.

If you would like to propose some documentation for this, you can make a PR at https://github.com/conda-forge/conda-forge.github.io/

**Comment by arturdaraujo** [comment]
I don't think this is the way to do it. It seems that the mamba install was removed. 

```
(base) artur@LAPTOP-80C5GA6H:~$ conda info

     active environment : base
    active env location : /home/artur/miniforge3
            shell level : 1
       user config file : /home/artur/.condarc
 populated config files : /home/artur/miniforge3/.condarc
          conda version : 24.9.2
    conda-build version : not installed
         python version : 3.10.14.final.0
                 solver : libmamba (default)
       virtual packages : __archspec=1=skylake
                          __conda=24.9.2=0
                          __cuda=12.6=0
                          __glibc=2.39=0
                          __linux=5.15.153.1=0
                          __unix=0=0
       base environment : /home/artur/miniforge3  (writable)
      conda av data dir : /home/artur/miniforge3/etc/conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
          package cache : /home/artur/miniforge3/pkgs
                          /home/artur/.conda/pkgs
       envs directories : /home/artur/miniforge3/envs
                          /home/artur/.conda/envs
               platform : linux-64
             user-agent : conda/24.9.2 requests/2.31.0 CPython/3.10.14 Linux/5.15.153.1-microsoft-standard-WSL2 ubuntu/24.04 glibc/2.39 solver/libmamba conda-libmamba-solver/24.1.0 libmambapy/1.5.8
                UID:GID : 1002:1002
             netrc file : None
           offline mode : False

```

**Comment by hmaarrfk** [comment]
> It seems that the mamba install was removed.

how does the info you showed show this?

**Comment by arturdaraujo** [comment]
The "mamba" command does not work anymore. I tried using `conda info` instead and the `mamba info`  and the mamba version (first line of the results) are not showing the mamba version.

**Comment by arturdaraujo** [comment]
I'm not entirely sure how Miniforge interacts with Conda, but it seems to work "on top" of Conda by adjusting some defaults and making a few other changes. I believe the update notifications come from the original Conda installation, not Miniforge. That’s why I looked to the documentation for clarification.

To update Miniforge, I usually do a fresh install because I'm unsure how to properly update it, which often results in keeping an outdated version for months. It would be helpful if the documentation included clear instructions on updating Miniforge (such as which commands to use in the base environment to update Mamba) or clarified that the Conda update messages don’t apply.

**Comment by hmaarrfk** [comment]
what does your .bashrc file contain?

**Comment by hmaarrfk** [comment]
(i run the suggested command when it tells me to, on ubuntu)

**Comment by arturdaraujo** [comment]
I ran `conda update -n base -c conda-forge conda` on my WSL Ubuntu with miniforge install. The Mamba version was 1.5.8 and now it doesn't even show which version of Mamba is installed anymore when I press `conda info` (because `mamba info` also doesn't work like it was removed or something) I try to create/delete/install packages with mamba command and it doesn't work. 

I did what was suggested and it is not working anymore, that's why I'm trying to clarify the problem. Maybe it has something to do with the version of the mamba (which was a bit old). As you said, you do the same on Ubuntu, and works.

It's probably due to the version of the mamba
